datasets/za/wanted/crawler.py

```diff
@@ -0,0 +1,61 @@
+from zavod import Context
+from lxml import html
+from urllib.parse import urlparse, parse_qs
+from typing import Dict
+import re
+
+from zavod import helpers as h
+
+REGEX_PATTERN = re.compile("(.+)\((.+)\)(.+)")
+
+
+def parse_cell(cell: html.HtmlElement):
+    result = {}
+    text = cell.text_content()
+    match = REGEX_PATTERN.match(text.strip())
+    if match:
+        name, crime, status = match.groups()
+        last_name, first_name = name.split(",", maxsplit=1)
+        result["first_name"] = first_name.strip()
+        result["last_name"] = last_name.strip()
+        result["crime"] = crime.strip()
+        result["status"] = status.strip()
+        result["source_url"] = cell.xpath(".//a/@href")[0]
+    return result
+
+
+def crawl_cell(context: Context, cell: Dict[str, str]):
+    data = parse_cell(cell)
+    
+    # first name is considered a bare minimum to emit a person entity
+    if data["last_name"] in ["Unknown", "Uknown"]:
+        return
+
+    person = context.make("Person")
+    
+    # each wanted person has a dedicated details page 
+    # which appears to be a unique identifier
+    id = parse_qs(urlparse(data["source_url"]).query)["bid"][0]
+    person.id = context.make_slug(id)
+
+    h.apply_name(
+        person,
+        first_name=data.get("first_name"),
+        last_name=data.get("last_name"),
```

Comment:
Sorry for the late request - could you do the logic in parse_cell inside crawl_cell (perhaps call it crawl_person)?

The thinking is just that it's kinda making a two step process what would be just as clear in one method, and that's closer to our usual style.

Reply:
this gives more of a one to one mapping between the source of the data and the destination.

Reply:
I've refactored `parse_cell` and `crawl_cell` into `crawl_person` on https://github.com/opensanctions/opensanctions/pull/533/commits/bf44b48987e110f070c394139fd3c6d7c8b2f4f9 and I think things look much clearer. Let me know if this looks good.